lib/src/frame.dart

```diff
@@ -22,6 +22,8 @@ final _v8Frame =
 
 // http://pub.dartlang.org/stuff.dart.js:560:28
 final _v8UrlLocation = new RegExp(r'^(.*):(\d+):(\d+)|native$');
+// eval at Foo (main.dart.js:588)
+final _v8UrlEvalLocation = new RegExp(r'^(.*):(\d+)');
```

Comment:
I'd prefer if we make the eval pattern below more accurate. That is, update _v8EvalLocation below to handle an optional column instead. As a result, the new logic would only apply on lines that contain `eval at`.

Reply:
@sigmundch i've updated. But regexp has no matches.
This is strange, because JS version of regexp handles matches: https://regex101.com/r/GLygY7/2

any issues with my regexp?

Reply:
I can't see the updated code in Dart. Is the commit uploaded?

Reply:
solution found. please check updated regexp